build/package/host/etc/cvdr.toml

```diff
@@ -25,6 +25,12 @@
 # files forever.
 # KeepLogFilesDays = 30
 
+# Source for the Build API credentials to be used on cvd create operations.
+# Possible values are:
+#  - "injected": Use credentials stored in the server.
+#  - "none": Don't use Build API credentials.
+# CredentialsSource = "injected"
```

Comment:
should we be more specific here: `BuildAPICredentialsSource`? 

First interpretation of CredentialsSource could be about credentials against the Cloud Orchestrator service itself, I could foresee users referencing this config if authentication against the CO fails.

Reply:
Done